react-basic-practice/src/themes/components/input.ts

```diff
@@ -0,0 +1,49 @@
+import { inputAnatomy as parts } from "@chakra-ui/anatomy";
+import { createMultiStyleConfigHelpers } from "@chakra-ui/styled-system";
+
+const { definePartsStyle, defineMultiStyleConfig } =
+  createMultiStyleConfigHelpers(parts.keys);
+
+const baseStyle = definePartsStyle({
+  field: {
+    border: "1px",
+    _placeholder: {
+      color: "riverbed",
+      fontSize: "md",
+      fontWeight: "normal",
+      lineHeight: "short",
+    },
+  },
+});
+
+const variantSearch = definePartsStyle({
+  field: {
+    padding: "0px 12px 0 36px",
+    height: "32px",
+    borderColor: "gullGray",
+    _placeholder: {
+      color: "gullGray",
+    },
+  },
+});
+
+const variantForm = definePartsStyle({
+  field: {
+    width: "342px",
+    height: "51px",
+    borderColor: "paleSky",
+  },
+});
+
+const variants = {
+  search: variantSearch,
+  form: variantForm,
+};
+
+export const inputTheme = defineMultiStyleConfig({
```

Comment:
`Input` is enough

Reply:
I implemented it according to the structure defined in the document https://v2.chakra-ui.com/docs/components/button/theming

Reply:
We can change it, right? It will help code cleaner